.github/workflows/build.yaml

```diff
@@ -0,0 +1,41 @@
+# https://github.com/teichae/github-action/blob/master/.github/workflows/actions.yml
+
+name: Build and Push
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v2.4.0
+        with:
+          images: mrchypark/generic-sidecar-injector
```

Comment:
I dont think that the main repo should have `mrchypark` here. Maybe use a variable/secret instead?

Reply:
I would appreciate it if you could kindly provide me with the necessary repository address. I do not have authorization for the Salesforce organization. The changes I made were done using my personal account based on my needs, so please make the appropriate modifications.